### DISCUSSION TITLE
How to change the type of the disk from simple to FCD?

### DISCUSSION THREAD

**Discussion by Cirilus** [vmware/govmomi#3398] [Community Help]
I have an simple disk that attached to VM, can I change the type of disk to FCD?

**Comment by dougm** [comment]
You should be able to use govc [disk.register ](https://github.com/vmware/govmomi/blob/main/govc/USAGE.md#diskregister)or the API it uses: https://github.com/vmware/govmomi/blob/007b3cd7cf343c4908c01bbddf9326b7cf9c239e/govc/disk/register.go#L63

**Reply by Cirilus** [reply]
Yeah, I used this method, but I had this error `ServerFaultCode: The request refers to an unexpected or unknown type.`. I tried to pass as the path: datastore path, path without datastore and url
